core/assets/bundles/bundle.properties

```diff
@@ -1128,6 +1129,7 @@ block.power-source.description = Infinitely outputs power. Sandbox only.
 block.item-source.description = Infinitely outputs items. Sandbox only.
 block.item-void.description = Destroys any items. Sandbox only.
 block.liquid-source.description = Infinitely outputs liquids. Sandbox only.
+block.liquid-void.description = Removes any liquids. Sandbox only.
```

Comment:
For French translation: `block.liquid-void.description = Détruit n'importe quel liquide. Bac à sable seulement.`.